jupyter_rfb/widget.py

```diff
@@ -317,11 +314,11 @@ def _rfb_send_frame(self, array, is_lossless_redraw=False):
         # Compose message and send
         msg = dict(
             type="framebufferdata",
-            src=src,
+            mimetype=mimetype,
             index=self._rfb_frame_index,
             timestamp=timestamp,
         )
-        self.send(msg)
+        self.send(msg, [data])
```

Comment:
Any particular reason for wrapping data in a list?

Reply:
This is just the Jupyter Widgets API. You could send multiple bytes in the same messages over the websocket:

```python
self.send({ type: "custom" }, [b'hello', b'world'])

```

- https://github.com/manzt/anywidget/issues/58#issuecomment-1419517506